apps/omg_child_chain/lib/omg_child_chain/application.ex

```diff
@@ -34,14 +35,31 @@ defmodule OMG.ChildChain.Application do
   end
 
   def start_phase(:attach_telemetry, :normal, _phase_args) do
+    config = %{
+      # DD reserved tag keys -- cannot use: host, device, service, source
+      # TODO(jacob) cleanup
+      tags: [
+        "application:#{Application.get_env(:omg_child_chain, :application)}",
+        "app_env:#{Application.get_env(:omg_child_chain, :app_env)}",
+        "contract_semver:#{Application.get_env(:omg_child_chain, :contract_semver)}",
+        "current_version:#{Application.get_env(:omg_child_chain, :current_version)}",
+        "env:#{Application.get_env(:omg_child_chain, :env)}",
+        "network:#{Application.get_env(:omg_child_chain, :network)}",
+        "release:#{Application.get_env(:omg_child_chain, :release)}",
+        # "service:#{Application.get_env(:omg_child_chain, :service)}",
+        "service_name:#{Application.get_env(:omg_child_chain, :service_name)}"
+      ]
+    }
+
     handlers = [
       [
         "measure-ethereum-event-listener",
         OMG.EthereumEventListener.Measure.supported_events(),
         &OMG.EthereumEventListener.Measure.handle_event/4,
         nil
       ],
-      ["measure-state", OMG.State.Measure.supported_events(), &OMG.State.Measure.handle_event/4, nil]
+      ["measure-state", OMG.State.Measure.supported_events(), &OMG.State.Measure.handle_event/4, nil],
+      Transaction.Metrics.events_handler(config)
```

Comment:
any reason not to stick to the convention?

Reply:
I looked over the code base and I didn't see a convention. I saw few a different ways it had been done. i didn't see any docs on it, but happy to follow the convention and also add docs if needed.

1. some metrics are called metrics and in dd they are metrics. these tend to be system-related metrics. ethereumex, tracers, 
2. some of the business-y stuff metrics are called measure. but they're actually metrics.
3. I wanted to encapsulated everything in the module eg: `Transaction.Metrics`. It sets the handler for transactions, all related telemetry event handlers and metric naming, tagging and publishing all from there one place. 

it would be a convention.

if you don't like it, give me 5 ticks and it's gone.